src/lib.rs

```diff
@@ -569,6 +577,7 @@ fn load_builtins(cache: &mut ProgramCacheForTxBatch) -> HashSet<Pubkey> {
     builtins.insert(solana_sdk::bpf_loader::id());
     builtins.insert(solana_sdk::bpf_loader_upgradeable::id());
     builtins.insert(solana_sdk::compute_budget::id());
+    builtins.insert(solana_sdk::loader_v4::id());
```

Comment:
this should be feature gated

Reply:
pretty sure it's handled with agave here: https://github.com/anza-xyz/agave/blob/024d047eedba22768099cc6509414202400c6231/runtime/src/bank/builtins/mod.rs#L126